### PR TITLE
【PPSCI Export&Infer No.3】euler_beam

### PR DESCRIPTION
**Pull request by GreatV**
### PR types
Others

### PR changes
Others

### Describe

- https://github.com/PaddlePaddle/PaddleScience/issues/788

### CODE REVIEW

File: examples/euler_beam/conf/euler_beam.yaml [hunk 1]
```diff
@@ -56,3 +58,21 @@ EVAL:
   pretrained_model_path: null
   eval_with_no_grad: true
   total_size: 100
+
+INFER:
+  pretrained_model_path: "https://paddle-org.bj.bcebos.com/paddlescience/models/euler_beam/euler_beam_pretrained.pdparams"
+  export_path: ./inference
```
Comment (HydrogenSulfate) [inline examples/euler_beam/conf/euler_beam.yaml:64]: `./inference `==> `./inference/euler_beam`

File: examples/euler_beam/conf/euler_beam.yaml [hunk 2]
```diff
@@ -56,3 +58,21 @@ EVAL:
...
+  pdmodel_path: ${INFER.export_path}.pdmodel
+  pdpiparams_path: ${INFER.export_path}.pdiparams
+  device: gpu
+  engine: native
+  precision: fp32
+  onnx_path: null
```
Comment (HydrogenSulfate) [inline examples/euler_beam/conf/euler_beam.yaml:70]: null ==> ${INFER.export_path}.onnx

File: examples/euler_beam/euler_beam.py [hunk 3]
```diff
@@ -219,14 +219,63 @@ def u_solution_func(out):
...
+        store_key: output_dict[infer_key]
+        for store_key, infer_key in zip(cfg.MODEL.output_keys, output_dict.keys())
+    }
+
+    ppsci.visualize.save_plot_from_1d_dict(
+        "./euler_beam_pred",
+        {**input_dict, **output_dict},
+        ("x",),
+        ("u",),
+    )
```
Comment (HydrogenSulfate) [inline examples/euler_beam/euler_beam.py:262]:
``` suggestion
def u_solution_func(out):
        """compute ground truth for u as label data"""
        x = out["x"]
        return -(x**4) / 24 + x**3 / 6 - x**2 / 4

    ppsci.visualize.save_plot_from_1d_dict(
        "./euler_beam_pred",
        {**input_dict, **output_dict, "u_label": u_solution_func(input_dict)},
        ("x",),
        ("u", "u_label"),
    )
```

**Review by HydrogenSulfate — CHANGES_REQUESTED** [review]
感谢大佬提交代码，几处细节辛苦修改一下。

File: examples/euler_beam/conf/euler_beam.yaml (hunk 2 above)
Comment (GreatV) [reply examples/euler_beam/conf/euler_beam.yaml]: Done.

File: examples/euler_beam/conf/euler_beam.yaml (hunk 1 above)
Comment (GreatV) [reply examples/euler_beam/conf/euler_beam.yaml]: Done.

**Review by HydrogenSulfate — APPROVED** [review]
LGTM

**Pull request merged**